EIPS/eip-5298.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 5298
+title: ENS as Token Holder
+description: An interface for holding tokens by ENS.
```

Comment:
This description is a bit vague, and is just a rewording of the title. Maybe choose a different word than "holding" here?

Reply:
I think so too, but didnt have a bette idea yet. Any suggestion? @SamWilsn